java/common/exception/ErrorMessage.java

```diff
@@ -62,17 +62,19 @@
     public static final ErrorMessage INVALID_RULE_WHEN_CONTAINS_DISJUNCTION =
             new ErrorMessage(21, "Rule '%s' 'when' contains a disjunction.");
     public static final ErrorMessage INVALID_RULE_THEN =
-            new ErrorMessage(22, "Rule '%s' 'then' must strictly be either an attribute ownership, or a relation: '%s'.");
+            new ErrorMessage(22, "Rule '%s' 'then' must strictly be a single of either an attribute ownership, or a relation");
```

Comment:
The sentence contains a mistake here: "must strictly be a single ..." ? Single "something" ?

Reply:
It means a single of either attribute ownership or a relation. In other words, either of those things, and exactly one instance of it. Reworded to make this clearer.